SAMtags.tex

```diff
@@ -353,21 +359,23 @@ \subsection{Annotation and Padding}
 %NOTE - This leaves open the possibility of allowing multiple such
 %entries for a single CT tag to be combined with | as in the PT tag.
 
-\item[PT:Z:\tagregex{\tt \emph{start};\emph{end};\emph{strand};\emph{type}(;\emph{key}(=\emph{value}))*(\char92|\emph{start};\emph{end};\emph{strand};\emph{type}(;\emph{key}(=\emph{value}))*)*}]
+\item[PT:Z:\tagregex{\metavar{annotag}(\char92|\metavar{annotag})*}]\enskip where each \metavar{annotag} matches\quad\tagregex{\metavar{start};\metavar{end};\metavar{strand};\metavar{type}(;\metavar{key}(=\metavar{value})?)*}
+\hfill\\
 Read annotations for parts of the padded read sequence.
 
-The {\tt PT} tag value has the format of a series of
+The {\tt PT} tag value has the format of a series of annotation
 tags separated by `{\tt |}', each annotating a sub-region of the read.
 Each tag consists of \emph{start}, \emph{end}, \emph{strand},
 \emph{type} and zero or more \emph{key}{\tt =}\emph{value} pairs, each
 separated with semicolons. \emph{Start} and \emph{end} are 1-based
 positions between one and the sum of the {\tt M/I/D/P/S/=/X}
 {\sf CIGAR} operators, i.e. {\sf SEQ} length plus any pads.  Note
-any editing of the CIGAR string may require updating the `{\tt PT}'
+any editing of the CIGAR string may require updating the {\tt PT}
 tag coordinates, or even invalidate them.
 As in GFF3, \emph{strand} is one of `{\tt +}' for forward strand tags,
```

Comment:
link to GFF3 format?

Reply:
Would you like to suggest a suitable link?

Reply:
the official, yet sparse:
https://useast.ensembl.org/info/website/upload/gff3.html

unofficial, but less sparse:
https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md

Reply:
I though the official version was at http://www.sequenceontology.org/ but they moved it to GitHub recently making https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md the official source now?